docs/backup-restore.md

```diff
@@ -1,3 +1,5 @@
+{% include end-of-support.html %}
```

Comment:
This link points https://github.com/scalar-labs/docs-scalardb/blob/main/_includes/end-of-support.html ?

Reply:
Correct. Sorry, I should have included a link. I've added it to my original comment.